cluster/saltbase/salt/supervisor/docker-checker.sh

```diff
@@ -25,7 +25,7 @@ echo "waiting a minute for startup"
 sleep 60
 
 while true; do
-  if ! sudo timeout 10 docker version > /dev/null; then
+  if ! sudo timeout 10 docker ps > /dev/null; then
```

Comment:
Can we measure the `docker ps` call latencies and determine the timeout accordingly? In some cases such as batch pod creation, `docker ps` could take longer than 10 seconds (albeit it should be rare). Also, we may want to retry here to make sure we don't kill docker based on one-off transient error.